README.md

```diff
@@ -1,4 +1,87 @@
-# Hello_Microverse-Pro
+# Hello-Microverse-Project
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+Hello-Microverse-Project
+
+> Describe your project in 1 or 2 sentences.
+Hello-Microverse-Project is a my first project after the enrollment with Microverse.This is the Project about basic setup of Repo in github with files such as README.md,.gitignore,linters.yml file and basic setup of HTML and CSS file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This Project is built with HTML and CSS.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Demo Project of Microverse]**
+- **[Getting familiar with Github,HTML,CSS]**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](http://127.0.0.1:5500/index.html)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+  cd my-folder
+  git clone https://github.com/pushkar1995/Hello-Microverse-Pro.git
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/pushkar1995)
+- Twitter: [@twitterhandle](https://twitter.com/Pushkar9951)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/pushkar-gautam-94a385271/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Will be adding more information regarding initial project of Microverse]**
+- [ ] **[Design of the project will be optimized]**
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my coding partners and microverse for making this project happen.
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
[OPTIONAL]

- [ ] Kindly delete any instructions since they were only for your guidance, leaving them makes your README looks unprofessional.